schoolopy/authentication.py

```diff
@@ -0,0 +1,135 @@
+import random
+import time
+import yaml
+import requests_oauthlib
+import json
+from urllib import parse
+import requests
+
+class SchoologyAuth:
+    def __init__(self, consumer_key, consumer_secret, user_id=None):
+        self.API_ROOT = 'https://api.schoology.com/v1'
+        self.DOMAIN_ROOT = 'https://fccps.schoology.com'
```

Comment:
This shouldn't be hardcoded.

Reply:
It would only work for us.

Reply:
Yeah I just put that in to test. I think we can add a prefix thing for people to enter.